src/harness/reference_models/pre_iap_filtering/pre_iap_filtering.py

```diff
@@ -0,0 +1,52 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""
+==================================================================================
+  This is the main Pre-IAP reference model which invokes the sub pre-IAP reference
+  models to filter out grants and CBSDs before IAP model is invoked.
+==================================================================================
+"""
+from reference_models.fss_purge import fss_purge
+from reference_models.inter_sas_duplicate_grant import inter_sas_duplicate_grant
+
+
+def performPreIapFiltering(protected_entities, sas_uut_fad, sas_test_harness_fads):
+  """ The main function that invokes all pre-IAP filtering models.
+
+  The grants/CBSDs to be purged are removed from the input parameters.
+  
+  Args:
+    protected_entities: A dictionary containing the list of protected entities. The key
+      is a protected enity type and the value is a list of corresponding protected
+      entity records. The format is {'entity_name':[record1, record2]}.
```

Comment:
In this doc string entity_name should be entityName to match the formatting of the keys.

Reply:
Corrected.